util/yaml/yaml_test.go

```diff
@@ -466,3 +466,26 @@ func TestFromUnstructured(t *testing.T) {
 	g.Expect(err).ToNot(HaveOccurred())
 	g.Expect(string(rawyaml)).To(Equal(string(convertedyaml)))
 }
+
+func TestRaw(t *testing.T) {
+	g := NewWithT(t)
+
+	input := "\n" +
+		"First Line" +
+		"\n" +
+		"Second Line"
+
+	output := "First Line\nSecond Line"
+	result := Raw(input)
+	g.Expect(result).To(Equal(output))
```

Comment:
I think you can drop this test case because our intent is to use this func only with multimedia - indented strings, like int the following test case

Reply:
Sure I will remove this test case.
Ya in the following commits I will use this function in test cases.